MetaMask/metamask-extension
# Fix #7073 - Provide a QR to export private key

## **Description**

It's been requested by many to provide a QR code to export private key in the "Show Private Key" flow.  This PR puts a QR code on the private key modal after the user has authenticated password and pressed-to-hold.

## **Related issues**

Fixes: #7073

## **Manual testing steps**

1. Click the three-dot global menu
2.  Click "Account Details"
3. Click "Show Private Key"
4. Enter correct password
5. Hold the button to reveal key
6. See the QR code above the text private key
7. Use a QR reader to read the value
8. Ensure the output matches the private key

## **Screenshots/Recordings**



### **Before**

N/A

### **After**

Not putting in the PR to not expose any data

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
